iotedgehubdev/__init__.py

```diff
@@ -8,6 +8,6 @@
 pkg_resources.declare_namespace(__name__)
 
 __author__ = 'Microsoft Corporation'
-__version__ = '0.14.7'
+__version__ = '0.14.8rc0'
```

Comment:
consider using 0.14.7rc1?

Reply:
@mhshami01 this is out of scope for 0.14.7